src/components/Requirements/SubRequirement.vue

```diff
@@ -107,17 +107,11 @@ import CompletedSubReqCourse from '@/components/Requirements/CompletedSubReqCour
 import IncompleteSubReqCourse from '@/components/Requirements/IncompleteSubReqCourse.vue';
 import DropDownArrow from '@/components/DropDownArrow.vue';
 
-import {
-  RequirementFulfillment,
-  EligibleCourses,
-  SubReqCourseSlot,
-  CrseInfo,
-} from '@/requirements/types';
+import { SubReqCourseSlot, CrseInfo } from '@/requirements/types';
 import { clickOutside } from '@/utilities';
 
 import { cornellCourseRosterCourseToFirebaseSemesterCourse } from '@/user-data-converter';
 
-Vue.component('completedsubreqcourse', CompletedSubReqCourse);
 Vue.component('incompletesubreqcourse', IncompleteSubReqCourse);
```

Comment:
Is there a blocker for migrating IncompleteSubReqCourse as was done for CompletedSubReqCourse?

Reply:
Yes, this component is still using these imported types:

https://github.com/cornell-dti/course-plan/blob/f046f281be638dbc9e494aa557acfbd8c32b21de/src/requirements/types.ts#L195-L205

I plan to investigate in another PR whether these types should be moved out to global types.